platform/podjs/src/browserPod.ts

```diff
@@ -354,6 +348,7 @@ class BrowserNetwork {
                 if (status < 200 || status > 299) {
                     fetchResponse.error = `Unexpected response: ${request.responseText}`;
                     resolve(fetchResponse);
+                    return;
```

Comment:
Isn't this an unrelated fix?

Reply:
When looking at Browserpod to add the https needed to fix this because it didn't work the other way (resolve doesn't stop code execution and all errors would be printed, which would lead to confusion). So yes and no